Framework/PythonInterface/CMakeLists.txt

```diff
@@ -10,8 +10,7 @@ set(Boost_LIBRARIES ${Boost_LIBRARIES} ${old_boost})
 
 # NumPy
 find_package(Numpy REQUIRED)
-include_directories(SYSTEM ${PYTHON_INCLUDE_PATH})
-include_directories(SYSTEM ${PYTHON_NUMPY_INCLUDE_DIR})
+include_directories(SYSTEM ${PYTHON_INCLUDE_PATH} ${PYTHON_NUMPY_INCLUDE_DIR} ${Boost_INCLUDE_DIRS})
```

Comment:
Is it worth switching these to `target_include_directories`?

Reply:
I think it's more complicated for PythonInterface; it is not a library but a custom target, I am not sure how to make this work with `target_include_directories`.